fastlane/lib/fastlane/actions/deploygate.rb

```diff
@@ -17,27 +13,44 @@ def self.is_supported?(platform)
         platform == :ios
       end
 
-      def self.run(options)
-        require 'shenzhen'
-        require 'shenzhen/plugins/deploygate'
+      def self.upload_build(api_token, user_name, ipa, options)
+        require 'faraday'
```

Comment:
Great point 👍 Should be a `fastlane` dependency. We now have 2 HTTP clients as dependency of _fastlane_, do you think we should get rid of one? (the other one)